update-version.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
```

Comment:
How is this file going to be triggered, manually?

Reply:
Yes, it's triggered manually. This is a helper script to simply automatically update the manifest after upstream releases an update.